lib/internal/streams/utils.js

```diff
@@ -262,15 +264,17 @@ function isErrored(stream) {
 }
 
 module.exports = {
-  kDestroyed,
+  isDestroyed,
+  kIsDestroyed,
   isDisturbed,
   kIsDisturbed,
   isErrored,
   kIsErrored,
   isReadable,
   kIsReadable,
+  isWritable,
+  kIsWritable,
```

Comment:
linter failure (although it'd be nice to keep both `kIsWritable` and `isWritable` together, GH won't let me send a suggestion for it.
```suggestion
  kIsWritable,
```